### PR TITLE
Header bug fix

### PR DESCRIPTION
**Pull request by Chebi97**

### CODE REVIEW

**Comment by dsusviela** [comment]
Header is now way better than what it used to be on mobile. Any chance we can have the hamburger button white instead of black?

Also, tablet view hasn't changed.

File: src/components/Profile/Profile.js [hunk 1]
```diff
@@ -27,8 +43,47 @@ class Profile extends Component {
...
+      ],
+      animated: false,
+    });
+  };
+
+  setButtonsPhone = () => {
+    this.props.navigator.setButtons({
+      rightButtons: [
+        {
+          icon: sideMenuIcon,
```
Comment (frabago) [inline src/components/Profile/Profile.js]: Should be white

**Comment by andresk727** [comment]
tablet error
In some cases user name doesn't appear in the header.

**Comment by andresk727** [comment]
It would be nice if the name also appeared when using mobile in landscape orientation.

**Pull request closed without merging**